secret/Tiltfile

```diff
@@ -89,6 +89,38 @@ def secret_from_dict(name, namespace="", inputs={}):
     args.extend(["-o=yaml", "--dry-run=client"])
     return local(args, quiet=True, echo_off=True)
 
+def registry_secret(name, namespace="", flags_dict={}):
+    """Returns YAML for a docker-registry secret
+    Args:
+        name: The configmap name.
+        namespace: The namespace.
+        inputs: A dict of keys and values to use. Nesting is not supported
+        flags_dict (optional): the flags used by docker-registry secret command
+          Example: '{"docker_server": "host", "docker_username": "username", "docker-password": "password", "docker_email": "email@email.com"}'
```

Comment:
i think you meant `docker-server` rather than `docker_server`? similarly for the other keys with underscores.

would it make more sense to hard-code the flags as arg params, like secret_yaml_tls does?